app/views/verify/review/new.html.slim

```diff
@@ -12,23 +12,23 @@ h1.h3.my0 = t('idv.titles.session.review')
     .pl5
       .h6 = t('idv.review.full_name')
       .h4.bold.ico-absolute.ico-absolute-success
-        = "#{idv_params[:first_name]} #{idv_params[:last_name]}"
+        = "#{@idv_params[:first_name]} #{@idv_params[:last_name]}"
 
       .mt3.h6 = t('idv.review.mailing_address')
       .h4.bold.ico-absolute.ico-absolute-success
-        | #{idv_params[:address1]}<br>
-        - if idv_params[:address2].present?
-          | #{idv_params[:address2]}<br>
-        | #{idv_params[:city]}, #{idv_params[:state]} #{idv_params[:zipcode]}
+        | #{@idv_params[:address1]}<br>
+        - if @idv_params[:address2].present?
```

Comment:
related but not, we do some very similar formatting + logic on the home/profile page to support addresses, I'm going to file a ticket to see if we can . refactor that into a partial?